nodejs/awsx/lb/application.ts

```diff
@@ -51,7 +51,7 @@ export class ApplicationLoadBalancer extends mod.LoadBalancer {
             }, opts)];
         }
 
-        super("aws:lb:ApplicationLoadBalancer", name, argsCopy,
+        super("awsx:lb:ApplicationLoadBalancer", name, argsCopy,
             pulumi.mergeOptions(opts, { aliases: [{ type: "awsx:x:elasticloadbalancingv2:ApplicationLoadBalancer"}] }));
```

Comment:
You'd have to do a type alias similar to what's done below already to ensure that resources aren't accidentally created/deleted.